components/insight/SRC/org/openmicroscopy/shoola/agents/metadata/editor/MapAnnotationsComponent.java

```diff
@@ -665,6 +665,7 @@ private void insertRow() {
 		MapTable t = getSelectedTable();
 		if (t == null)
 			t = getUserTable();
+		if (t == null) return;
```

Comment:
Since the same check is performed one line above, should we combine both blocks?